ca/certificate-authority.go

```diff
@@ -214,6 +216,14 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		return emptyCert, err
 	}
 
+	fmt.Println(ca.NotAfter)
```

Comment:
Remove printlns. Also, this should be a config variable, since the length of time a cert is valid for a is configurable. That config currently lives in cfssl. We could duplicate it to the Boulder config.

Another thing occurred to me - does cfssl already do this check?